service/src/main/java/uk/gov/mca/beacons/api/beacon/mappers/BeaconMapper.java

```diff
@@ -79,6 +82,7 @@ public BeaconRegistrationDTO toRegistrationDTO(
       .lastServicedDate(beacon.getLastServicedDate())
       .mti(beacon.getMti())
       .svdr(beacon.getSvdr())
+      .mod(ModEmail != null)
```

Comment:
This should just be beacon.mod

As it's stored in the db we just need to pull it out